api/urls.py

```diff
@@ -11,4 +11,7 @@
     ),
     path("login/", TokenObtainPairView.as_view(), name="token_obtain_pair"),
     path("token/refresh/", TokenRefreshView.as_view(), name="token_refresh"),
+    path("api/schema/", SpectacularAPIView.as_view(), name="schema"),
+    path("stage/tasks/<int:project_id>/<int:stage_id>", views.TaskList.as_view(), name="stage_tasks"),
```

Comment:
E đặt lại tên resource path nhé
Tham khảo convention về đặt tên resource https://restfulapi.net/resource-naming/
```suggestion
    path("projects/<int:project_id>stages/<int:stage_id>/tasks", views.TaskList.as_view(), name="stage_tasks"),
```